pydra/engine/helpers_file.py

```diff
@@ -139,8 +139,11 @@ def hash_dir(
         for filename in filenames:
             if ignore_hidden_files and filename.startswith("."):
                 continue
-            this_hash = hash_file(dpath / filename)
-            file_hashes.append(this_hash)
+            if not is_existing_file(dpath / filename):
+                file_hashes.append(str(filename))
```

Comment:
shouldn't this be `str(dpath / filename)`

Reply:
Changed!